src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -192,6 +193,11 @@ function validate_options_type_by_value(input_options_with_data) {
             const details = `type of flag ${option} should be ${type_of_option} (and the received value is ${value})`;
             throw_cli_error(ManageCLIError.InvalidArgumentType, details);
         }
+        // special case for unset value (specified by '').
+        if (!LIST_UNSETTABLE_OPTIONS.includes(option) && value === '') {
+            const details = `flag ${option} is '' ${value} but it can't be unset.`;
```

Comment:
The `${value}`is '', and you already printed it, I'm not sure you need it.
```suggestion
            const details = `flag value of ${option} is '' but this option can't be unset.`;
```